apollo-router/src/uplink/testdata/schema_enforcement_directive_arg_version_in_range.graphql

```diff
@@ -4,6 +4,51 @@ schema
   query: Query
 }
 
+
+directive @join__enumValue(graph: join__Graph!) repeatable on ENUM_VALUE
```

Comment:
is that (and other schema changes) to convert to fed 2?

Reply:
No, previously some of these files had just enough Federation boilerplate to be able to parse as AST and go through `license_enforcement.rs`. Now they have enough to validate as well, but fed 1 vs fed 2 is unchanged.